src/AppKit/NSFont.cs

```diff
@@ -149,10 +153,13 @@ public static NSFont ControlContentFontOfSize (nfloat fontSize)
 			return ptr == IntPtr.Zero ? null : new NSFont (ptr);
 		}
  
-#if !NET
- 		[Deprecated (PlatformName.MacOSX, 10, 13)]
+#if NET
+ 		[UnsupportedOSPlatform ("macos10.13")]
+#if MONOMAC
+ 		[Obsolete ("Starting with macos10.13.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#endif
 #else
-		[UnsupportedOSPlatform ("macos10.13")]
+ 		[Deprecated (PlatformName.MacOSX, 10, 13)]
```

Comment:
Nitpick: if we're changing source code, we might as well fix the indentation as well to not have spaces... but this is not important at all